uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/StaffItem.kt

```diff
@@ -0,0 +1,86 @@
+package io.github.droidkaigi.feeder.staff
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import io.github.droidkaigi.feeder.Staff
+import io.github.droidkaigi.feeder.core.NetworkImage
+import io.github.droidkaigi.feeder.core.theme.ConferenceAppFeederTheme
+import io.github.droidkaigi.feeder.fakeStaffs
+
+@Composable
+fun StaffItem(staff: Staff, onClickItem: (Staff) -> Unit) {
+
+    ConstraintLayout(
+        modifier = Modifier
+            .semantics { testTag = "StaffItem" }
+            .clickable {
+                onClickItem(staff)
+            }
+            .fillMaxWidth()
+            .semantics(mergeDescendants = true) { }
+            .padding(start = 8.dp, end = 8.dp),
+    ) {
+        val (image, name) = createRefs()
+
+        Surface(
+            modifier = Modifier
+                .constrainAs(image) {
+                    top.linkTo(parent.top, 8.dp)
+                    start.linkTo(parent.start, 16.dp)
+                    bottom.linkTo(parent.bottom, 16.dp)
+                }
+                .size(56.dp),
+            shape = CircleShape
+        ) {
+            NetworkImage(
+                url = staff.image,
+                modifier = Modifier.fillMaxSize(),
+                contentScale = ContentScale.Fit,
+                contentDescription = "Staff Icon"
+            )
+        }
+        Text(
```

Comment:
The design comp shows a name and a title for each staff, but the `Staff` model doesn't have the title text. `StaffApi` doesn't have it either, so I made it only display the name. I assume the design is outdated, but let me know if I'm wrong.

Reply:
Thank you for handling this problem.
It's OK as it is!